src/main/java/com/hydratereminder/HydrateReminderConfig.java

```diff
@@ -151,6 +151,24 @@ default boolean hydrateReminderChatMessageEnabled()
 		return true;
 	}
 
+	/**
+	 * <p>Allows the player to set the type of his personality based on which he will receive personalized messages
+	 * </p>
+	 * @return the type of personality
+	 * @since 2.0.0
+	 */
+	@ConfigItem(
+			keyName = "hydrateReminderPersonalityType",
+			name = "Personality type",
+			description = "Sets the type of personality by which different messages will be displayed when hydrating",
+			section = HYDRATE_REMINDER_NOTIFICATIONS_SECTION,
+			position = +1
+	)
+	default HydrateReminderPersonalityType hydrateReminderPersonalityType()
+	{
+		return HydrateReminderPersonalityType.FUN;
+	}
+
```

Comment:
@SaranshBaniyal As mentioned in #221, the position of this setting should be below the `Computer notification` setting.

![image](https://user-images.githubusercontent.com/1442227/196625453-a2e84625-66d4-4ce3-8023-e8e188c377a5.png)

So two things need to be changed here:

1. The value of `position` should be 1 greater than the position of `Computer notification`
2. The placement of this config item code should be placed under the `Computer notification` config item code

Reply:
@jmakhack sure. I have pushed another commit, please have a look